server/chat-commands/info.ts

```diff
@@ -2644,6 +2645,27 @@ export const commands: ChatCommands = {
 	},
 	showhelp: [`/show [url] - shows an image or video url in chat. Requires: whitelist % @ # &`],
 
+	regdate: 'registertime',
+	regtime: 'registertime',
+	async registertime(target, room, user) {
+		this.checkChat();
+		this.runBroadcast();
+		if (!user.autoconfirmed) return this.errorReply(`Only autoconfirmed users can use this command.`);
+		target = toID(target);
+		if (!target) target = user.id;
+		const raw = await Net(`https://${Config.routes.root}/users/${target}.json`).get().catch(() => {});
+		if (!raw) return this.sendReplyBox(`The user '${target}' is unregistered.`);
```

Comment:
Is that really the only possible error here? I think we should handle network failures.

Reply:
It's the most common failure, and people would panic at network failures, so imo i don't think we need to scare people?

Reply:
For the record, not getting a response from the server means that the login server is probably down (How were you using the command in the first place).

For example the unregistered account ``Zarel is cute`` would give 
```{"username":"zareliscute","userid":"zareliscute","registertime":0,"group":0,"ratings":{}}```

It doesn't return any error.  It returns actual JSON.  I did say that if ``userdata.registertime === 0`` then it's unregistered before....

Also regarding the JSON.parse(), I'm not sure how the user json api for the login server works, but if the server really was down, it's possible you might get some html code (for error 500) instead of JSON.  That would cause an actual crash because JSON.parse won't work.

((Did you actually test it and use the command on an unregistered account?))

Reply:
Yes, i did test it on a non-regged ac- that's where the error came from. 
I do not get `{"username":"name","userid":"name","registertime":0,"group":0,"ratings":{}}` as a response.

Reply:
The error is `not found`.

Reply:
I don't think the correct solution to "people would panic at network failures" is "we should lie to them".

...to be clear I'm saying you should detect the 404 condition and say "User is unregistered", differently from other errors which should say "Something went wrong with the request" (and probably just forward the original error message), maybe with "try again later" in there.

Reply:
Ok.